app/controllers/v0/vic/vic_submissions_controller.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module V0
+  module VIC
+    class VICSubmissionsController < ApplicationController
+      skip_before_action(:authenticate)
+
+      def create
+        vic_submission = ::VIC::VICSubmission.new(
+          params.require(:vic_submission).permit(:form)
+        )
+        vic_submission.user_uuid = current_user.uuid if current_user.present?
+
+        unless vic_submission.save
```

Comment:
Should this be `save!`?

Reply:
i raise on line 19 after logging to sentry so don't need to raise on the save